src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/errors/BadRequest.kt

```diff
@@ -2,5 +2,8 @@ package org.vaccineimpact.orderlyweb.errors
 
 import org.vaccineimpact.orderlyweb.models.ErrorInfo
 
-class BadRequest(message: String)
-    : OrderlyWebError(400, listOf(ErrorInfo("bad-request", message)))
+class BadRequest(messages: List<String>)
```

Comment:
name of this class could be stating the obvious but naming as `BadRequest` is confusing.

Reply:
This is a pre-existing class, not added in this PR. What do you think would be better than `BadRequest`? `BadRequestError` or `BadRequestException`? The 'BadRequest' bit does make sense since it's returning a 400. In general, the names of the exception classes are fairly inconsistent - some are 'Error's, some are 'Exception's, some are neither.. I don't really see the value of changing this in this PR, but could add a maintenance ticket to make the exception classes' names more consistent / readable in general.